src/components/home/PanelResult.vue

```diff
@@ -203,40 +219,44 @@ export default Vue.extend({
   computed: {
     ...mapGetters({
       userGenes: 'getUserGenes',
-      userGenesInPanels: 'getUserGenesInPanels',
       institutionMap: 'getInstitutionMap',
+      panelSearchResult: 'getPanelSearchResult',
+      parsedGenes: 'getParsedGenes'
     }),
   },
+  watch: {
+    panelSearchResult: 'showPanelResult',
+  },
   methods: {
-    getPanelResult() {
-      return this.userGenesInPanels.map((panel: PanelSearchResult) => {
+    showPanelResult() {
+      this.notFoundGenes = this.parsedGenes.notFoundGenes
+      this.panelContent = this.panelSearchResult.map((panel: PanelSearchResult) => {
         const genesInPanel = panel.genesInPanel.map((gene: Gene) =>
-          gene.name.toUpperCase()
+            gene.name.toUpperCase()
         )
         const genesNotInPanel = panel.genesNotInPanel.map((gene: Gene) =>
-          gene.name.toUpperCase()
+            gene.name.toUpperCase()
         )
-
-        let institution = this.institutionMap.get(panel.name)
+        let institution = this.institutionMap.get(panel.name.toUpperCase())
         if (!institution) {
-          institution = {}
+          institution = {};
         }
 
         return new PanelResultFormattedRow(
-          panel.name,
-          genesInPanel.length,
-          genesNotInPanel.length,
-          genesInPanel,
-          genesNotInPanel,
-          institution
+            panel.name,
+            genesInPanel.length,
+            genesNotInPanel.length,
+            genesInPanel,
+            genesNotInPanel,
+            institution
```

Comment:
Something wrong with indentation here (and in some other places). It should be 2 instead of 4.

Reply:
i will check

Reply:
done